test/parallel/test-dgram-send-callback-buffer.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+
+var fs = require('fs');
+var dgram = require('dgram');
+var callbacks = 0;
+var client, timer, buf;
+
+client = dgram.createSocket('udp4');
+
+buf = new Buffer(256);
+
+client.send(buf, common.PORT, '127.0.0.1', function(err, bytes) {
+  assert.equal(bytes, buf.length);
+  clearTimeout(timer);
+  client.close();
+});
+
+timer = setTimeout(function() {
+  throw new Error('Timeout');
+}, 200);
```

Comment:
I copied it over from the other dgram tests (the same applies to `common.mustCall`).
The whole purpose it making sure that the callback is called within a definite timeframe. We can move this to 1 or 2 seconds without issues.

Should I submit a different PR for some test cleanup for all of dgram tests?